File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerDashboardResponse.java
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto;

import java.math.BigDecimal;
import java.util.UUID;

public record SellerDashboardResponse(
        UUID sellerId,
        String sellerName,
        String sellerIntro,
        BigDecimal totalRevenue,
        Long totalOrderCount,
        BigDecimal refundRate,
        BigDecimal confirmedRate) {
}

File: src/main/kotlin/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerDashboardResponse.kt
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto

import java.math.BigDecimal
import java.util.*

data class SellerDashboardResponse(
    val sellerId: UUID,
    val sellerName: String,
    val sellerIntro: String?,
    val totalRevenue: BigDecimal?,
    val totalOrderCount: Long?,
    val refundRate: BigDecimal?,
    val confirmedRate: BigDecimal?
)
